qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -1035,6 +1024,49 @@ def get_parameter_value(
         # 5) Return the most recent parameter.
         return max(enumerate(candidates), key=lambda x: (x[1].date_time, x[0]))[1].value
 
+    def _standardize_assign_params(
+        self,
+        assign_params: Dict,
+        qubits: Tuple[int, ...],
+        schedule_name: str,
+    ) -> Dict[ParameterKey, ParameterValueType]:
+        """Standardize the format of manually specified parameter assignments.
+
+        Users specify parameter assignment dictionaries as tuples. This function (a) converts
+        these tuples to ``ParameterKey`` instances and (b) looks up parameter links between
+        schedules and adds them to the standardized returned parameter assignment dictionary.
+
+        Args:
+            assign_params: The dictionary that specifies how to assign parameters.
+            qubits: The qubits for which to get a schedule.
+            schedule_name: The name of the schedule in which the parameters can be found.
+
+        Returns:
+            A dict with :class:`.ParameterKey`s as keys and :class:`.ParameterValueType`s as values.
+        """
+        assign_params_, linked_assign_params = dict(), dict()
+
+        if assign_params:
+            for param, value in assign_params.items():
+                if isinstance(param, str):
+                    assign_params_[ParameterKey(param, qubits, schedule_name)] = value
+                else:
+                    assign_params_[ParameterKey(*param)] = value
+
+            # Add parameter links for automatic linking.
+            for key, value in assign_params_.items():
```

Comment:
```suggestion
        if assign_params:
            # Add parameter links for automatic linking.
            for param, value in assign_params.items():
                if isinstance(key, str):
                    key = ParameterKey(param, qubits, schedule_name)
                else:
                    key = ParameterKey(*param)
```
regarding my [comment](https://github.com/Qiskit/qiskit-experiments/pull/889/files#r960412852) above